contributed_definitions/NXmpes.nxdl.xml

```diff
@@ -128,20 +158,27 @@
                 <field name="fast_axes" type="NX_CHAR" recommended="true"/>
                 <field name="slow_axes" recommended="true"/>
                 <group type="NXcollectioncolumn">
+                    <!--
+                        TODO: Comment from Anders Frisk on proposal page
+                        What is the definition of a collection column?
+
+                        Optional constant settings (like lens focusing parameters on the sample: position_x, position_y, working_distance)
+                    -->
```

Comment:
We can extend the docstring in the base class. I don't understand the question for constant settings. Values are per definition not part of an application definition or base class. Working_distance does exist, no?

Reply:
I think this question rather relates to how are information between different parts of the instrument are collected, i.e., how do we know the distance to the sample if only information in the collection column is written from the vendor.
I think it can be properly answered with NXtransformations (which is also a solution to having a more modular geometry).

Reply:
Has anything been done here?

Reply:
I don't think so. I agree that in principle these fixed parameters (position_x, position_y) can be given by using the existing `NXcollection_column/NXtransformations`, so there shouldn't be a need for adding anything else here.